src/client/menu/repositories/components/add-repo-button/index.js

```diff
@@ -1,59 +0,0 @@
-import React from 'react'
-import CSSModules from 'react-css-modules'
-
-import { func } from 'prop-types'
-
-import Button from '../../../components/button'
-
-import css from './index.css'
```

Comment:
Not super important by any means, but the local imports could be grouped together.

Reply:
This file was deleted =P